internal/app/connectconformance/testsuites/data/basic.yaml

```diff
@@ -172,7 +174,7 @@ testCases:
     - "@type": type.googleapis.com/connectrpc.conformance.v1.BidiStreamRequest
 # Misc Tests ------------------------------------------------------------------
 - request:
-    testName: unimplemented
+    testName: unary/unimplemented
```

Comment:
I don't think this one needs a "unary" prefix since the stream type is really irrelevant to what it's testing.